Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -82,19 +95,14 @@ spec:
       - ip: 192.168.39.182
 ```
 
-You can use the existing `object-external.yaml` file.
-When ready the ceph-object-controller will output a message in the Operator log similar to this one:
+You can use the existing `object-external.yaml` file. Even though multiple endpoints can be specified, but it is recommend to use only one endpoint.
+If there are multiple endpoints, please add load balancer in front of them and use the load balancer endpoint in the `externalRgwEndpoints` list.
 
 ```console
-ceph-object-controller: ceph object store gateway service >running at 10.100.28.138:8080
-```
-
-You can now get and access the store via:
+When ready the ceph-object-controller will output a message in the Operator log similar to this one:
 
 ```console
-$ kubectl -n rook-ceph get svc -l app=rook-ceph-rgw
-NAME                     TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
-rook-ceph-rgw-my-store   ClusterIP   10.100.28.138   <none>        8080/TCP   6h59m
+ceph-object-controller: ceph object store gateway service >running at 10.100.28.138:8080
```

Comment:
Will the external RGW address be put into the OBC configmap?

Reply:
Yes, if there are multiple then it will be randomly assigned. It was fixed in https://github.com/rook/rook/pull/11202